Games/Retro_ML.SuperMarioWorld/Neural/Scoring/TimeTakenScoreFactor.cs

```diff
@@ -42,6 +42,8 @@ public TimeTakenScoreFactor()
 
         public string Name => "Time taken";
 
+        public string Tooltip => "Gives points when the AI takes way too long to complete a level. Recommended to set to a negative value and leave enabled in case AIs decide to take way too much time on a level";
```

Comment:
```suggestion
        public string Tooltip => "Reward applied when the AI takes way too long to complete a level. Recommended to set to a negative value and leave enabled in case AIs decide to take way too much time on a level";
```